worlds/shorthike/docs/setup_en.md

```diff
@@ -0,0 +1,33 @@
+# A Short Hike Multiworld Setup Guide
+
+## Required Software
+
+- A Short Hike: [Steam](https://store.steampowered.com/app/1055540/A_Short_Hike/)
+    - The Epic Games version of A Short Hike will also work.
+- A Short Hike Modding Tools: [GitHub](https://github.com/BrandenEK/AShortHike.ModdingTools)
+- A Short Hike Randomizer: [GitHub](https://github.com/BrandenEK/AShortHike.Randomizer)
+
+## Optional Software
+
+- [PopTracker](https://github.com/black-sliver/PopTracker/)
+  - [Chandler's A Short Hike PopTracker Pack](https://github.com/chandler05/shorthike-archipelago-poptracker/releases)
+
+## Installation
+
+1. Download the [Modding Tools](https://github.com/BrandenEK/AShortHike.ModdingTools/releases), and follow
+the [installation instructions](https://github.com/BrandenEK/AShortHike.ModdingTools#a-short-hike-modding-tools) on the GitHub page.
+
+2. After the Modding Tools have been installed, download the 
+[Randomizer](https://github.com/BrandenEK/AShortHike.Randomizer/releases) and 
+archive and extract the contents of both
+into the `Modding` folder.
+
+## Connecting
+
+A Short Hike will prompt you with the server details when a new game is started or a previous one is continued.
+Enter in the Server Port, Name, and Password in the popup menu that appears and hit connect.
```

Comment:
```suggestion
Enter in the Server Port, Name, and Password (optional) in the popup menu that appears and hit connect.
```
Have seen more than a few people confused about passwords in the past, adding (optional) here should make it more clear to some that if you don't have a password, you don't need to put anything there.